spec/services/doc_auth/acuant/pii_from_doc_spec.rb

```diff
@@ -19,7 +19,7 @@
         state_id_expiration: '2022-10-24',
         state_id_number: 'DOE-84-1165',
         state_id_jurisdiction: 'ND',
-        state_id_type: 'drivers_license',
+        state_id_type: 'state_id_card',
```

Comment:
Nice how this work helped spot and fix a preexisting issue 👍 

https://github.com/18F/identity-idp/blob/e5b72af77401c00508cefabad56072ca054923e8/spec/fixtures/acuant_responses/get_results_response_success.json#L1267-L1271